### PR TITLE
TMHS Cusp3D to TitanSMA SM recorder swap

### PR DESCRIPTION
**Pull request by Thomas-Benson**
- Removing Cusp3D (42032) from TMHS LC 21
- Replacing with TitanSMA (002209) at same location
- Using latest TitanSMA firmware version 4.5.22

### CODE REVIEW

File: install/recorders.csv [hunk 1]
```diff
@@ -1485,6 +1485,7 @@ Nanometrics,Titan,TitanSMA,002203,PNMS,21,283,,0,0,2024-02-29T02:18:00Z,9999-01-
 Nanometrics,Titan,TitanSMA,002204,FXBS,20,145,,0,0,2024-02-28T23:26:00Z,9999-01-01T00:00:00Z
 Nanometrics,Titan,TitanSMA,002205,WKZ,21,0,,0,0,2024-02-20T21:06:00Z,9999-01-01T00:00:00Z
 Nanometrics,Titan,TitanSMA,002208,WBHS,21,0,,0,0,2024-02-28T23:23:00Z,9999-01-01T00:00:00Z
+Nanometrics,Titan,TitanSMA,002209,TMHS,21,0,,0,0,2024-05-30T23:32:00Z,9999-01-01T00:00:00Z
```
Comment (gva6500) [inline install/recorders.csv:1488]: think its missing the new azimuth  of 250 here...

Comment (Thomas-Benson) [reply install/recorders.csv]: Good catch, fixed

**Review by mnaguit — COMMENTED** [review]
Thanks Thomas, please include the CUSP3D (42032) in firmware.

**Comment by Thomas-Benson** [comment]
🤦 I always forget CUSPs are the only older recorders we use to actually have firmware entries. Anyway should be fixed now @mnaguit

**Review by CBurton90 — APPROVED** [review]

**Comment by mnaguit** [comment]
Hi Thomas, this looks good, thanks. Just a quick tweak for the uppercase letter -- new entry in firmware for Nanometrics.

**Comment by Thomas-Benson** [comment]
Fixed the firmware `Make`, hopefully that's the last thing I missed

**Review by gva6500 — COMMENTED** [review]
lgtm

**Review by mnaguit — APPROVED** [review]
Thank you Thomas! 👍

**Review by CBurton90 — APPROVED** [review]

**Pull request merged**